index.js

```diff
@@ -2,5 +2,10 @@
 'use strict';
 
 module.exports = {
-  name: 'ember-devtools'
+  name: 'ember-devtools',
+
+  isEnabled: function isEnabled() {
+    var addonConfig = this.app.project.config(this.app.env)['ember-devtools'];
```

Comment:
I don't believe so, this is how it is done in the Ember CLI codebase, as well as several other addons I looked at.